uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -177,6 +177,12 @@ public void performSharedDeviceRegistration(String username, String password) {
         }
     }
 
+    @Override
+    public void performSharedDeviceRegistrationDontValidate(@NonNull final String username,
+                                                            @NonNull final String password) {
+        throw new UnsupportedOperationException("Not Supported!");
```

Comment:
Can you implement this here in Broker Host, as we have the control of it's UI,

Reply:
links to the test cases https://identitydivision.visualstudio.com/DevEx/_workitems/edit/833517 & https://identitydivision.visualstudio.com/DevEx/_workitems/edit/833511